src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -163,8 +163,8 @@ protected function __construct($result)
         $this->clean = $result['clean'];
         $this->digitised = ($result['digitised'] == 't') ? true : false;
         $this->digitisedby = empty($result['digitisedby']) ? null : (int) $result['digitisedby'];
-        $this->lastedited = empty($result['lastedited']) ? null : $result['lastedited'];
-        $this->lasteditedby = empty($result['lasteditedby']) ? null : (int) $result['lasteditedby'];
+        $this->last_edited_time = empty($result['last_edited_time']) ? null : $result['last_edited_time'];
```

Comment:
if you're converting it into a timestamp to go into the db, then do you need to convert here to store in the object?